cmd/cmd.go

```diff
@@ -49,8 +49,8 @@ func AddFlags(cmd *cobra.Command) {
 
 	cmd.PersistentFlags().StringP(FlagLogLevel, "L", "info",
 		"Set the log level")
-	cmd.PersistentFlags().String(FlagLogFile, "",
-		"Set the log file path. If not set, logs will output to stdout")
+	cmd.PersistentFlags().String(FlagLogFile, "/tmp/br.log",
```

Comment:
It is best to output each backup to a separate log file, and please use `os.TempDir()`

The following method may be useful to you.
```
func TimestampLogFileName() string {
	return fmt.Sprintf("%s/br-%s", os.TempDir(), time.Now().Format(time.RFC3339))
}
```